.github/ISSUE_TEMPLATE/2_feature_request_form.yml

```diff
@@ -0,0 +1,44 @@
+name: Feature request
+description: Suggest an idea for this project
+title: "[Feature Request]: "
+labels: ["triage needed", "enhancement"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to report a bug in the Oppia project.
+  - type: markdown
+    attributes:
+      value: |
+        Before filing a new issue, **please do a quick search** to check that it hasn't already been filed on the issue tracker. You can do this by going to [this link](https://github.com/oppia/oppia-android/issues) and typing some words related to the issue in the search box next to the "New issue" button.
+  - type: textarea
+    id: problem
```

Comment:
Why is it that the textareas of bug reports have an id, but the feature request forms don't?

Reply:
That's a good question -- I'm not sure, I copied the feature template from the Oppia Web version. The [docs](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository) suggest having an id (though I'm not sure what it's used for), so I've gone ahead and added IDs to the bug report form as well. Thanks for catching!

Reply:
I did some digging around and found that these may be used in a scenario like the one Ben described in #5043